components/EditorOverview.vue

```diff
@@ -77,22 +77,47 @@ function playDemoClicked(){
      * contenteditable element which is not presents on a page in a moment of button click
      *
      * So we use fake invisible input to focus it first, then change focus to our editor
+     * 
+     * setTimeout is used because the virtual keyboard interrupts scroll
```

Comment:
This problem and solution is unclear to me. Do you mean mobile devices?

For now, it breaks the focus of editor on mobile devices (tested in Safari), which is working fine in `main`. Also, scroll works fine in `main` on mobile safari

Reply:
Are you sure because I tested it on Safari (IOS Simulator), it was working fine, and yes the focus and scroll breaks on android.

Reply:
Another solution is to put
`const block: HTMLElement | null = document.querySelector('.ce-paragraph');

      if (block !== null){

        block.focus()
      }`
inside onReady callback which is woking on Chrome (Android) and Safari(IOS Simulator).

Reply:
I have kept the changes to a minimum this PR only addresses the scroll issue on web tested on Chrome (Desktop) and Safari (Desktop)